src/components/SelectionList/Search/UserInfoCell.tsx

```diff
@@ -5,15 +5,18 @@ import Text from '@components/Text';
 import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useThemeStyles from '@hooks/useThemeStyles';
 import {isCorrectSearchUserName} from '@libs/SearchUIUtils';
+import type {AvatarSizeName} from '@styles/utils';
 import CONST from '@src/CONST';
+import type {PersonalDetails} from '@src/types/onyx';
 import type {SearchPersonalDetails} from '@src/types/onyx/SearchResults';
 
 type UserInfoCellProps = {
-    participant: SearchPersonalDetails;
+    participant: SearchPersonalDetails | PersonalDetails;
```

Comment:
clean up these types - they make the component be connected to 2 different types of persona data.

Let's just make it depend either:
 - on specific fields needed inside directly
 - some simple Person-like structure that only contains those fields